doc/nrf/releases_and_maturity/migration/migration_3.1_54h_suit_ironside.rst

```diff
@@ -0,0 +1,103 @@
+
+* You have a working |NCS| 3.0 application for the nRF54H20 SoC using SUIT.
+* You have installed |NCS| 3.1 and its toolchain.
+
+Breaking Changes
+================
+
+The following is a summary of the breaking changes that apply when migrating applications:
+
+* SUIT support is removed in |NCS| 3.1.
+  The Secure Domain now runs IronSide SE, which uses MCUboot for device firmware update.
+* IronSide SE uses a new UICR format and moves peripheral configuration into a dedicated MRAM partition.
+* Shared memory (IPC) is relocated to a fixed RAM-20 region.
+  You no longer reserve it manually.
+* DFU support moves from SUIT to MCUboot.
+  You must enable and configure MCUboot in your project.
+
+Update prj.conf
+---------------
+
+Remove SUIT-specific Kconfig options:
+
+* ``CONFIG_SUIT``
+* ``CONFIG_SUIT_SECURE_DOMAIN``
+
+Enable MCUboot for DFU:
+
+* ``CONFIG_MCUBOOT=y``
+* ``CONFIG_BOOTLOADER_MCUBOOT=y``
+
+Update Device Tree (DTS)
+------------------------
+
+Remove the old UICR partition
+    In your board's DTS overlay, remove any node that defined the ``uicr`` partition.
+
+Add IronSide peripconf partition
+    IronSide expects a blob of address-value pairs called ``peripconf`` stored in MRAM.
+
+Remove IPC-shared-memory reservation
+    As IronSide relocates the IPC buffer to a fixed RAM-20 address, you can delete any manual reservation in RAM-0.
+
+Peripheral configuration (peripconf)
+------------------------------------
+
+The new UICR format no longer holds peripheral initial values.
+You must generate a peripconf blob at build time.
+
+* Zephyr build applies an ``nrf_rpc_peripconf`` script.
+* The script reads ``peripconf`` nodes in the DTS.
+* It outputs a binary blob and embeds it into the MRAM partition.
+
+You do not need to modify your application code.
+You only need to ensure the DTS partition exists.
+
+Memory protection and isolation
+-------------------------------
+
+* IronSide SE currently grants full memory-access permissions to both application and radio domains by default.
+* You should delete any UICR settings related to the following:
+
+    * IPC location
+    * Secure-Domain offsets
+    * partition lock bits
+
+* Future releases will enable per-partition isolation via Kconfig (`CONFIG_IRONSIDE_MPC_CONF`) and additional DTS nodes.
+
+Memory map changes
+------------------
+
+The following memory map updates apply when migrating to IronSide SE:
+
+* The application core firmware now always starts at address ``0xE03_0000``, which is the first address in MRAM00 immediately following the IronSide firmware.
+* The radio core is not started automatically.
+* Nordic-reserved partitions in ``MRAM10`` and ``RAM0x`` have been removed.
+* IPC buffers toward the Secure Domain are relocated to fixed addresses in ``RAM20``.
+  Memory previously reserved in ``RAM0x`` for IPC can now be repurposed.
+* The devicetree no longer uses the ``nordic,owned-memory`` or ``nordic,owned-partitions`` compatibles.
```

Comment:
Maybe add a small hint to move all partition definitions directly under the `mram1x` partitions node?